### PR TITLE
Update: New List Demo Page with Updated Documentation

### PR DESCRIPTION
**Pull request by simeonoff**
Should be merged after this [Samples Pull](https://github.com/IgniteUI/igniteui-angular-samples/pull/30) is merged and deployed.

### CODE REVIEW

File: en/components/list.md [hunk 1]
```diff
@@ -5,71 +5,368 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
...
+})
+export class AppModule {}
+```
+
+Next, we need to add some more information to our contact object, like a `photo` source for our avatar and a `isFavorite` property to indicate the contact's favorite status.
+
+```typescript
+// contacts.comoponent.ts
+
+private contacts = [{
```
Comment (rkaraivanov) [inline en/components/list.md]: Change it to `public`. 
Any decorated class members and data bound properties that are not `public` will currently break AOT builds

File: en/components/list.md [hunk 2]
```diff
@@ -5,71 +5,368 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
...
+        imports: [..., IgxFilterModule]
+    })
+
+    // contacts.component.ts
+    ...
+    import { IgxFilterOptions } from 'igniteui-js-blocks/main';
+
+    @Component({...})
+    export class ContactListComponent {
+        private searchContact: string;
```
Comment (rkaraivanov) [inline en/components/list.md]: Same as above.

File: en/components/list.md [hunk 3]
```diff
@@ -5,71 +5,368 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
...
-#### List Properties
-- `children` - Array of all `IListChild` components: items and headers
-- `items` - Array of items in the list
-- `headers` - Array of headers in the list
-- `allowLeftPanning` - Determines whether the left panning of an item is allowed
-- `allowRightPanning` - Determines whether the right panning on an item is allowed
+```typescript
+// contacts.component.ts
+...
+private contacts = [{
```
Comment (rkaraivanov) [inline en/components/list.md]: Same as below.

**Comment by kdinev** [comment]
@simeonoff I guess @ig-ealbert should translated the changes before merging?

File: en/components/list.md [hunk 4]
```diff
@@ -5,71 +5,368 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
...
+
 <p class="highlight">The Ignite UI for Angular List component displays rows of items and supports one or more header items as well as search and filtering of list items. Each list item is completely templateable and will support any valid HTML or Angular component. </p>
 <div class="divider"></div>
 
 ### List Demo
-<div class="sample-container" style="height: 760px">
-<iframe src='https://{environment:host}/angular-demos/list' width="100%" height="100%" seamless frameBorder="0"></inframe>
+
+<div class="sample-container" style="height: 477px">
+<iframe src='http://{environment:host}/list-sample-4' width="100%" height="100%" seamless frameBorder="0"></inframe>
```
Comment (kdinev) [inline en/components/list.md]: `</inframe>` should be `</iframe>`. There's other occurrences down the document.

**Comment by ig-ealbert** [comment]
@kdinev @simeonoff Just the Grid needs to be translated again?  Or Grid and List?

File: en/components/list.md [hunk 5]
```diff
@@ -5,71 +5,368 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
...
+In this article we covered a lot of ground with the list component. We created a list of contact items. Used some additional Ignite UI for Angular components inside our list items, like avatars and icons. Created some custom item layout and styled it. Finally, we added list filtering. The list component has a few more APIs to explore, which are listed below.
+
+####Inputs
+The following inputs are available on the **igx-list** component:
+| Name | Type | Description |
+| :--- | :--- | :--- |
+| `emptyListImage` | String | Provide an image url to be used as placeholder when the list is empty. |
+| `emptyListMessage` | String | Provide an emtpy list message. |
+| `emptyListButtonText` | String | Customize the empty list button text. |
+| `emptyListButtonText` | String | Customize the empty list button text. |
```
Comment (ig-ealbert) [inline en/components/list.md]: Why is the same API listed twice? `emptyListButtonText`  Also, "empty" is misspelled on line 329 as "emtpy".

File: en/components/list.md [hunk 6]
```diff
@@ -5,71 +5,368 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
...
+
+The following inputs are available on the **igx-list-item** component:
+| Name | Type | Description |
+| :--- | :--- | :--- |
+| `isHeader` | Boolean | Sets the current list items as list header. |
+
+<div class="divider"></div>
+
+####Outputs
+The following outputs are availabe on the **igx-list** component:
```
Comment (ig-ealbert) [inline en/components/list.md]: available is misspelled.

File: en/components/list.md [hunk 7]
```diff
@@ -5,71 +5,368 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
...
+| Name | Type | Description |
+| :--- | :--- | :--- |
+| `emptyListButtonClick` | EventEmitter | Emmits an event with the current list when the Empty List Button is clicked. |
```
Comment (ig-ealbert) [inline en/components/list.md]: Emits is misspelled.

File: en/components/list.md [hunk 8]
```diff
@@ -5,71 +5,368 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
...
+
+###Additional Resources
 
-#### Item Properties
-- `index` - The index of item in children array
-- `hidden` - Determines whether the item should be displayed
-- `panState` - Gets the item's pan state
-- `options` - Defines the options of the particular list item that will be displayed on item swipe (pan)
 <div class="divider--half"></div>
+Our comminity is active and always welcoming to new ideas.
```
Comment (ig-ealbert) [inline en/components/list.md]: Community is misspelled.

**Comment by simeonoff** [comment]
@ig-ealbert Just the list.

**Comment by ig-ealbert** [comment]
@simeonoff Thanks!  I translated it yesterday and it is awaiting review by my team.  Once it is reviewed I will update the pull request with the new version of the file.

**Comment by kdinev** [comment]
@simeonoff Can you put the `contributing.md` under a `.github` folder?

**Comment by ig-ealbert** [comment]
I have submitted a pull request to @simeonoff that contains the translated list.md file.
https://github.com/simeonoff/igniteui-docfx/pull/1/files

**Pull request merged**